Tutorials/start/hello-world.md

```diff
@@ -1,68 +1,72 @@
 # Hello World
 
-With hardware connected as well as Proxy configured and running you are ready to write your first Python script to command hardware using the API! We're going to communicate through Proxy over the USB cable with the CC430 and turn the LED's ON and OFF. Let's go!
+With hardware connected, hardware configured, and Proxy running you are ready to write your first Python script to command hardware using the API! We're going to communicate through Proxy over the USB cable with the CC430 and turn the LED's ON and OFF. It's our very own "[hello-world](https://en.wikipedia.org/wiki/%22Hello,_World!%22_program)". Let's go!
 
 > NOTE: All commands are checked for corruption by the CC430 prior to accepting them but they are not currently acknowledged or guaranteed to be received.
 
-###Prerequisites
- * Faraday radio connected via USB to the computer
- * Proxy configured and running
- 
+## Prerequisites
+* Faraday radio connected via USB to the computer
+* Faraday radio configured correctly
+* Proxy configured and running
+* Download [hello-world.py](https://github.com/kb1lqc/Faraday-Software/raw/issue220/Tutorials/start/configuring-proxy.md) to you computer, open in a text editor
```

Comment:
ACTION @kb1lqc : Uhhh you have the WRONG link here. Sould be `...kb1lqc/Faraday-Software/issue220/Tutorials/start/hello-world.py`